docs/ems_reporting.md

```diff
@@ -0,0 +1,27 @@
+# EMS reporting
+
+Cumulus reports usage statistics to the [ESDIS Metrics System (EMS)](https://earthdata.nasa.gov/about/science-system-description/eosdis-components/esdis-metrics-system-ems).
+Two types of reports are generated: ingest and distribution.
+
+## Ingest
+
+TODO - add content here
```

Comment:
Maybe just get a quick blurb from @jennyhliu on this so we can just wrap this stuff in. i.e. we have 3 reports, this is what they are, this is what you configure. Doesn't have to be in depth.